doc/classes/AudioServer.xml

```diff
@@ -35,6 +35,26 @@
 				Generates an [AudioBusLayout] using the available buses and effects.
 			</description>
 		</method>
+		<method name="get_absolute_time" qualifiers="const" experimental="">
+			<return type="float" />
+			<description>
+				Returns the absolute time in seconds of the [AudioServer] timeline based on the number of audio frames mixed. Used to schedule sounds to be played in the future with higher accuracy.
+				[b]Note:[/b] This value only updates each time an audio chunk is mixed and should not be relied on as an accurate "current" time of the [AudioServer].
+				[b]Example:[/b] Schedule two sounds to be played at the same time ~1 second in the future:
```

Comment:
It's mostly because the tilde is rarely used in the class reference. Not that big of a deal though, feel free to disregard.
```suggestion
				[b]Example:[/b] Schedule two sounds to be played at the same time, roughly 1 second in the future:
```

Reply:
I think it's worth changing just to keep things consistent, thanks!